src/vm/win32threadpool.cpp

```diff
@@ -421,7 +421,20 @@ BOOL ThreadpoolMgr::Initialize()
     // initialize Worker and CP thread settings
     DWORD forceMin;
     forceMin = GetForceMinWorkerThreadsValue();
-    MinLimitTotalWorkerThreads = forceMin > 0 ? (LONG)forceMin : (LONG)NumberOfProcessors;
+
+    if (forceMin <= 0)
+    {
+        forceMin = NumberOfProcessors;
+
+#ifdef FEATURE_PAL	
+        UINT cpuLimit;	
+
+        if (PAL_GetCpuLimit(&cpuLimit) && cpuLimit < forceMin)	
+            forceMin = cpuLimit;	
+#endif
+    }
+
+    MinLimitTotalWorkerThreads = forceMin;
```

Comment:
Does https://github.com/dotnet/coreclr/blob/4f4bed705c14133731a852b27e8f412fe20efc2c/src/vm/win32threadpool.cpp#L626 also need to be updated?

Reply:
I think that one is ok. User can set whatever MinWorkerThreads is needed - between 1 and MaxPossible. 
I.E. It can be 1 or 1000 because user may have reasons.